client/extensions/woocommerce/state/ui/shipping/zones/methods/actions.js

```diff
@@ -33,32 +64,29 @@ export const removeMethodFromShippingZone = ( siteId, methodId ) => {
  * Changes the given shipping method into a shipping method of a new type. Note that, underneath, this removes the
```

Comment:
Change the comment here to say that it will change the method currently being edited.